### ISSUE TITLE
Check correctnes of help instructions

### ISSUE THREAD

**Issue by crtahlin** [jmozah/intOS-dfs#62]
`dff >>>> help `
outputs:

```
...
- copyFromLocal <source file in local fs, destination directory in pod, block size in MB>
...

```

But putting `1`as the blocksize argument goes through uploading more than 80.000 blocks (after which it failed, but not sure if I didn't kill the process by mistake). While putting `1000` uploads 19170 blocks. The file size is roughly 15 MB. So mybe these are bytes, not Mbytes?

Also, the filename length is limited to 50 chars? Should be enlarged and/or the limit mentioned in help.

**Comment by jmozah** [comment]
fixed in commit df4933ace06e76874a548dfc7724dfbfb4763304

**Comment by crtahlin** [comment]
verified
